l10n_es_ticketbai/models/account_tax.py

```diff
@@ -24,6 +24,9 @@ def tbai_is_subject_to_tax(self):
         return self not in s_iva_ns_taxes
 
     def tbai_is_tax_exempted(self):
+        isp_no_exempted = self.env.ref("l10n_es.1_account_tax_template_s_iva0_isp").id
```

Comment:
Ojo porque estás comparando un "account.tax.template" con un "account.tax". Entiendo que quieres comprar el impuesto creado por la template  "l10n_es.1_account_tax_template_s_iva0_isp" para la compañía correspondiente ¿correcto?

Reply:
Si efectivamente, estaba viendo claro siempre he tirado de metadatos por interfaz. ¿podéis decirme como debería ser por favor?

Reply:
Tenéis un método en l10n_es_aeat para obtener el impuesto de una compañía a partir del impuesto plantilla.

Reply:
Entiendo que pasando la plantilla aquí self.company_id.get_taxes_from_templates('l10n_es.1_account_tax_template_s_iva0_isp') me devuelve los impuestos generados y luego ya solo es comprobar el id del impuesto con el resultado de esa función "ids"

Reply:
No, debes usar `self.company_id.get_taxes_from_templates(self.env.ref('l10n_es.account_tax_template_s_iva0_isp'))`

El 1 es el identificador de compañía.

Reply:
exacto!!